Assets/BossRoom/Scripts/Client/Game/Action/TargetActionFX.cs

```diff
@@ -22,7 +22,7 @@ public TargetActionFX(ref ActionRequestData data, ClientCharacterVisualization p
         public override bool Start()
         {
             base.Start();
-            m_ParentState = m_Parent.Parent.GetComponent<NetworkCharacterState>();
+            m_ParentState = m_Parent.transform.parent.GetComponent<NetworkCharacterState>();
```

Comment:
We could cache a reference to the parent's `NetworkCharacterState`, and remove parent entirely, since it seems like it's deprecated (removes two GetComponents).